nemo_skills/inference/generate.py

```diff
@@ -520,8 +508,25 @@ async def process_single_datapoint(self, data_point, all_data):
             if self.cfg.override_max_code_executions and self.cfg.total_code_executions_in_prompt is not None:
                 generation_params["max_code_executions"] = data_point["total_code_executions"]
 
+        # Tracking the tokens and generation time
+        input_sequence_length = None
+        if self.prompt is not None:
+            if generation_params["prompt"] is not None:
```

Comment:
we should ideally still calculate this statistic as we still have access to tokenizer even if prompt isn't defined